packages/core/http/core-http-server/src/router/route.ts

```diff
@@ -63,6 +63,11 @@ export interface RouteConfigOptionsBody {
    */
   accepts?: RouteContentType | RouteContentType[] | string | string[];
 
+  /**
+   * A mime type string overriding the 'Content-Type' header value received.
+   */
+  override?: string | undefined;
```

Comment:
Mind elaborating a bit on that addition and the actual behavior of that option? The TSDoc isn't helping much 😅 

Also NIT: You'll likely never explicitly pass `undefined`, right? so `override?: string` may be sufficient?

Reply:
As far as TS is concerned both types are identical. `override?: string` gets translated to `override?: string | undefined` so it's just more explicit. Happy to change it though.